app/assets/stylesheets/defaults.scss

```diff
@@ -0,0 +1,11 @@
+nav {

```

Comment:
It should be requiring fine? but I can add it there. Worried about order?

Reply:
Fixed with 82f911e

Reply:
Yeah, the order is important here, because we may have specificity conflicts.  `require_tree` has undefined ordering, so I'd like to avoid future problems by taking it on here.